aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -860,8 +836,8 @@ func TestCloudFrontStructure_expandGeoRestriction_whitelist(t *testing.T) {
 	if *gr.Quantity != 3 {
 		t.Fatalf("Expected Quantity to be 3, got %v", *gr.Quantity)
 	}
-	if !reflect.DeepEqual(gr.Items, aws.StringSlice([]string{"CA", "GB", "US"})) {
-		t.Fatalf("Expected Items be [CA, GB, US], got %v", gr.Items)
+	if !reflect.DeepEqual(aws.StringValueSlice(gr.Items), []string{"GB", "US", "CA"}) {
```

Comment:
The previous resource logic sorted the slice, which is no longer the case for TypeSet usage.